README.md

```diff
@@ -19,6 +19,35 @@ Alternatively, build it with `cargo build --release`, which places the binary in
 
 Run the program with the `--help` flag to see all supported arguments.
 
+```shell
+$ cargo run --release -- --help
+Extract article HTML from Wikipedia Enterprise HTML dumps.
+
+Expects an uncompressed dump connected to stdin.
+
+Usage: om-wikiparser [OPTIONS] <OUTPUT_DIR>
+
+Arguments:
+  <OUTPUT_DIR>
+          Directory to write the extracted articles to
+
+Options:
+      --wikidata-ids <WIKIDATA_IDS>
+          File of Wikidata QIDs to extract, one per line (e.g. `Q12345`)
```

Comment:
I wrongly assumed that WIKIDATA_IDS should contain "one per line...".
Can it be reworded? E.g. "Path to file that contains one per line Wikidata QID (e.g. `Q12345`)"